.ci/Jenkinsfile

```diff
@@ -378,7 +379,13 @@ def generateFunctionalTestStep(Map args = [:]){
                         powershell label: "Check for running kibana", script: "Test-NetConnection ${linuxIp} -Port 5601"
                         powershell label: "Check for running fleet server", script: "Test-NetConnection ${linuxIp} -Port 8220"
                         cmd script: "go get -v github.com/cucumber/godog/cmd/godog@v0.11.0", label: "Download godog 0.11"
-                        cmd script: "cd e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.tags=\"${tags}\"", label: "Run functional tests for ${platform}:${suite}:${tags}"
+                        // This step will help to send the APM traces to the
+                        // APM service defined by the Otel Jenkins plugin.
+                        withOtelEnv() {
+                          dir("e2e\\_suites\\${suite}") {
+                            cmd(script: "make --no-print-directory functional-test", label: "Run functional tests for ${platform}:${suite}:${tags}")
```

Comment:
Is make working on windows?

Reply:
🤞 , theoretically there is chocolatey recipe for that -> https://community.chocolatey.org/packages/make 

But I'm not sure if it's available in the CI Windows workers, I'm giving a go blindly